canvas_modules/harness/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we change the code to only have 1 babel configuration?  There is already on under the scripts folder.